app/public/styles/rct/custom/tables/flags.css

```diff
@@ -63,8 +63,12 @@
 }
 
 /* Verification time */
-.rct .flags-table .flags-ver_time-cell,
-.rct .flags-table .flags-ver_time-header {
+.rct .flags-table .flags-verification_time-cell,
+.rct .flags-table .flags-verification_time-header {
```

Comment:
Now you are mixing kebab-case with snake_case

Reply:
Yes, I just follow design introduced some time ago, some css styles are applied column-name-wise, so if a columns has identifier (name) in snake case the css styles as well, if you look on other files you will se the same